tests/e2e/utils/PreferencesHandler.ts

```diff
@@ -15,19 +15,32 @@ export class PreferencesHandler {
      */
     public async setTerminalType(type: string) {
         Logger.debug('PreferencesHandler.setTerminalToDom');
+        await this.setPreference('terminal.integrated.rendererType', type);
+    }
+
+    /**
+     *
+     * @param askForConfirmation possible values are "never", "ifRequired" and "always"
+     */
+    public async setConfirmExit(askForConfirmation: string) {
```

Comment:
Does it make sense to change argument type to enum instead "string"?
I mean that in case with "string" it is not understandable what possible variants exist.
Instead enum makes it more clear:
```
enum ExitConfirmation {
    NEVER = 'never'
    ...
}

 public async setConfirmExit(askForConfirmation: ExitConfirmation)
```
Also, I think it will be also helpful for terminal type setting.
What do you think about that?

Reply:
Actually, I thought about enum too when I was writing it. But then I created a description that is shown when you want to use that.
![Screenshot from 2020-02-26 14-15-29](https://user-images.githubusercontent.com/29278800/75348161-8fe02880-58a2-11ea-8a41-8049794efd29.png)
Do you think Enum would be more useful? If so, I would still like to merge that as it is and fix that in the following PR - if you are OK with that.

Reply:
For me, enum looks more unambiguously. But it was just a question, and not mandatory to implement. Your variant also looks good.